deployment/terraform/coordinator.go

```diff
@@ -47,32 +44,35 @@ func (t *Terraform) StartCoordinator() error {
 		return err
 	}
 
+	agentConfig, err := t.generateLoadtestAgentConfig(output)
+	if err != nil {
+		return err
+	}
+	data, err := json.MarshalIndent(agentConfig, "", "  ")
+	if err != nil {
+		return err
+	}
+	dstPath := "/home/ubuntu/mattermost-load-test-ng/config/config.json"
+	mlog.Info("Uploading updated load-test agent config file")
+	if out, err := sshc.Upload(bytes.NewReader(data), dstPath, false); err != nil {
+		return fmt.Errorf("error uploading file, output: %q: %w", out, err)
+	}
+
 	mlog.Info("Setting up coordinator", mlog.String("ip", ip))
-	clusterConfig := coordinator.Config{
-		ClusterConfig: cluster.LoadAgentClusterConfig{
-			Agents:         loadAgentConfigs,
-			MaxActiveUsers: 100,
-		},
-		MonitorConfig: performance.MonitorConfig{
-			PrometheusURL:    "http://" + output.MetricsServer.Value.PrivateIP + ":9090",
-			UpdateIntervalMs: 2000,
-			Queries: []prometheus.Query{
-				{
-					Description: "Request Duration",
-					Query:       "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
-					Threshold:   2.0,
-					Alert:       true,
-				},
-			},
-		},
-	}
-
-	data, err := json.MarshalIndent(clusterConfig, "", "  ")
+
+	coordinatorConfig, err := coordinator.ReadConfig("")
```

Comment:
There is no doubt an improvement here. However, -since we are here-, this is still a hardcoding. Maybe we can supply a config path to the `ltctl loadtest start` command as `--coordinator-config`? And use it's default path?

Reply:
Yeah. As I said above, we want to do that but we then need to pass the load-test config too. We end up potentially passing 3 different config files to `ltctl`. I want to address this but I opted for keeping it in a separate PR so we have more time to properly think about an optimal solution.